### DISCUSSION TITLE
Is it possible to configure Tasmota Devices, WITHOUT getting the Power shut down?

### DISCUSSION THREAD

**Discussion by fellpower** [arendst/Tasmota#17734] [Q&A]
hey

i have my PC on a Nous A1T plug. Now i want to change some config on it, but it always restart and kill the power of my pc, when im writing the config.

Is it possible, to get that relais on, when restarting, so my pc wont get off, when updating some conf on it?

**Comment by barbudor** [comment]
Depending what you are changing on the configuration, Tasmota will restart.
That happens when changing GPIO configurations, network and mqtt settings and a few other features that are initialized only at bot time.

However, unless you expressely changing the configuration of the relay (like changing GPIO, or changing between Relay and Relay Inverted) or have especially instructed Tasmota to switch off the relay on restart, there is no reason why the relay would be switched off.
It would also be a retained POWER OFF MQTT message.

So in clear, you have something wrong in your setup and without more details it is not possible to tell you exactly what

1st question does the relay switch off also if you manually initiate a restart with command 'Restart 1' or from the red button on the GUI ?
2nd question do you ffel the switch off occurs when Tasmota is shuting down or on reboot ?

You need to get logs with a log level of 3 or 4 to better understand what happens
Web console might not be very usefull so I suggest to use instead Syslog or MqttLog

**Comment by fellpower** [comment]
is it clear. i have this power plug - and if im changing some configuration, like mqtt server, that plug restarts and the relay goes off. but at this plug is my pc, is its going off too. ALL my tasmota plugs do that, from the factory. 

so, can i configure that devices, that they dont power off the plug (and my oc on it), if they restart?

**Reply by s-hadinger** [reply]
It shouldn't change the state of the relay when you restart. Like @barbudor said, something is wrong with your setup

**Reply by sfromis** [reply]
With other Tasmota plug-like devices, I'm routinely upgrading Tasmota settings and new firmware versions (restarts in both cases) without power being cut. What Tasmota version are you on? Do you have a setting for PowerOnState? Do you have retained messages possibly kicking in?

Do follow the suggestion from barbudor about getting logs.

**Comment by fellpower** [comment]
Power on state is 3

I tried that with last tasmota fw 12.3.1 on an esp8266 based nous A1T plug. and i tried 8 of them, and all doing that. loosing the 230V power when restarting.

how do i fix that?

**Reply by sfromis** [reply]
While not very similar, with version 12.3.1 there has been an issue with some devices having a power blink when they should not. Try upgrading to latest development build from http://ota.tasmota.com/tasmota/
